app/models/businessmatching/BusinessActivities.scala

```diff
@@ -185,23 +193,25 @@ object BusinessActivities {
   def getValue(ba:BusinessActivity): String =
     ba match {
       case AccountancyServices => "01"
-      case BillPaymentServices => "02"
-      case EstateAgentBusinessService => "03"
-      case HighValueDealing => "04"
-      case MoneyServiceBusiness => "05"
-      case TrustAndCompanyServices => "06"
-      case TelephonePaymentService => "07"
+      case ArtMarketParticipant => "02"
```

Comment:
Changes back to 02?

Reply:
Natural ordering in model / form, backwards-compatible ordering in JSON